benchopt/plotting/plot_objective_curve.py

```diff
@@ -145,6 +160,13 @@ def compute_quantiles(df_filtered):
     return q1, q9
 
 
+def reset_solvers_idx():
```

Comment:
Could you add a doc string.

For the name, I would say: `reset_solver_styles_idx` to make it clear what the idx represent?